.github/workflows/on-pre-release.yaml

```diff
@@ -0,0 +1,29 @@
+name: Pre-Release CI
+
+on:
+  push:
+    tags:
+      - 'v*.*.*-rc.*'
+
+jobs:
+  run-unit-tests:
+    name: Run Unit Tests
+    uses: ./.github/workflows/test.yaml
+
+  publish-docker-image:
+    name: Publish Pre-Release Docker Image
+    uses: ./.github/workflows/publish-docker-images.yaml
+    secrets: inherit
+    if: endsWith(github.event.release.tag_name, '-rc.*')
```

Comment:
Is this duplicated with tags matching?

Reply:
Yepp, it is. Pushing to remove duplicate